std/digest/digest.d

```diff
@@ -579,6 +579,15 @@ interface Digest
         @trusted nothrow void put(scope const(ubyte)[] data...);
 
         /**
+         * This is a conveniance function to pass some strings with char
+         * as element encoding type.
+         */
+        @trusted nothrow final void putText(scope const(char)[] data...)
```

Comment:
Perhaps this is a hijacking issue? I don't see why it shouldn't work, but if it doesn't, In your class, put:
```D
alias I.put put;
```
This should pull in the base names to be properly overloaded.